src/f5_tts/infer/utils_infer.py

```diff
@@ -5,33 +5,35 @@
 
 sys.path.append(f"../../{os.path.dirname(os.path.abspath(__file__))}/third_party/BigVGAN/")
 
+from f5_tts.model.utils import (
+    get_tokenizer,
+    convert_char_to_pinyin,
+)
+from f5_tts.model import CFM
+from vocos import Vocos
+from transformers import pipeline
+from pydub import AudioSegment, silence
+import tqdm
+import torchaudio
+import torch
+import numpy as np
+import matplotlib.pylab as plt
 import hashlib
 import re
 import tempfile
 from importlib.resources import files
+from pathlib import Path
+from huggingface_hub import snapshot_download
 
 import matplotlib
 
 matplotlib.use("Agg")
 
-import matplotlib.pylab as plt
-import numpy as np
-import torch
-import torchaudio
-import tqdm
-from pydub import AudioSegment, silence
-from transformers import pipeline
-from vocos import Vocos
-
-from f5_tts.model import CFM
-from f5_tts.model.utils import (
-    get_tokenizer,
-    convert_char_to_pinyin,
-)
 
 _ref_audio_cache = {}
 
-device = "cuda" if torch.cuda.is_available() else "mps" if torch.backends.mps.is_available() else "cpu"
+device = "cuda" if torch.cuda.is_available(
```

Comment:
will need pre-commit formatting,
refer to https://github.com/SWivid/F5-TTS?tab=readme-ov-file#development
same for following code

p.s. just need run the pre-commit once, before `git commit`

Reply:
I was having some formatting issues I guess with it:

```
src\f5_tts\model\modules.py:746:18: F722 Syntax error in forward annotation: `b n d`
    |
744 |     def __call__(
745 |         self,
746 |         x: float["b n d"],
    |                  ^^^^^^^ F722
747 |         text_embed: float["b n d"],
748 |     ):
    |
    
   ```
   
   which were present already in the repo before.  I committed without verification, but it reformatted the repo, got any suggestions?

Reply:
similar as in current modules.py file, you could do 
`# noqa: F722`

Reply:
Thanks, never use pre-commit before.  Looks like something I'll get used to using, good tool.